Tests/Marketplace/search_and_install_packs.py

```diff
@@ -224,43 +295,48 @@ def install_packs(client, host, prints_manager, thread_index, packs_to_install,
-        prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN, include_timestamp=True)
-        prints_manager.execute_thread_prints(thread_index)
+        else:
+            request_data = {
+                'packs': packs_to_install,
+                'ignoreWarnings': True
+            }
 
-        # make the pack installation request
-        try:
-            response_data, status_code, _ = demisto_client.generic_request_func(client,
-                                                                                path='/contentpacks/marketplace/install',
-                                                                                method='POST',
-                                                                                body=request_data,
-                                                                                accept='application/json',
-                                                                                _request_timeout=request_timeout)
+            packs_to_install_str = ', '.join([pack['id'] for pack in packs_to_install])
+            message = 'Installing the following packs in server {}:\n{}'.format(host, packs_to_install_str)
+            prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN,
+                                         include_timestamp=True)
+            prints_manager.execute_thread_prints(thread_index)
 
-            if 200 <= status_code < 300:
-                message = 'Packs were successfully installed!\n'
-                prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN,
-                                             include_timestamp=True)
-            else:
-                result_object = ast.literal_eval(response_data)
-                message = result_object.get('message', '')
-                err_msg = f'Failed to install packs - with status code {status_code}\n{message}\n'
+            # make the pack installation request
+            try:
+                response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                                    path='/contentpacks/marketplace/install',
+                                                                                    method='POST',
+                                                                                    body=request_data,
+                                                                                    accept='application/json',
+                                                                                    _request_timeout=request_timeout)
+
+                if 200 <= status_code < 300:
+                    message = 'Packs were successfully installed!\n'
+                    prints_manager.add_print_job(message, print_color, thread_index, LOG_COLORS.GREEN,
```

Comment:
While you're in it please print the response here too.

And use `pformat` to print the tables (see example number 4 in the screenshot attached.

![image](https://user-images.githubusercontent.com/41257953/97837301-9dc63300-1ce6-11eb-8604-027d631d4fd8.png)

Reply:
Sure about this? @hod-alpert 
It looks really bad, since we have 10 instances running, its making a lot of prints, lots on noise

Reply:
Yes yes, it's an important print, I'll make it look better in the future